EIPS/eip-draft_bls_keygen.md

```diff
@@ -0,0 +1,280 @@
+---
+eip: <to be assigned>
```

Comment:
Please use 2333 (this PR) or 2337 (the issue number). The same rule applies to your other submissions :wink: